static/app/views/performance/landing/views/frontendOtherView.tsx

```diff
@@ -14,7 +14,7 @@ export function FrontendOtherView(props: BasePerformanceViewProps) {
     <PerformanceDisplayProvider
       value={{performanceType: PROJECT_PERFORMANCE_TYPE.FRONTEND_OTHER}}
     >
-      <div data-test-id="frontend-pageload-view">
```

Comment:
Is this no longer needed?

Reply:
wrong data-test-id for frontend-other, not using it in a test yet so I yoinked it.